install.hs

```diff
@@ -1,8 +1,7 @@
 #!/usr/bin/env stack
 {- stack
-  --stack-yaml shake.yaml
-  --install-ghc
-  runghc
+  script
+  --resolver nightly-2018-12-17
```

Comment:
resolver should not be equal to a resolver in one of the `stack-$VER.yaml`'s. See #1116 and the respective fix #1137.
tldr, I request changing the resolver to `nightly-2018-12-15`

Reply:
Done!